From 9a8120a8d7eb872da43e61d598c226f0d6b7ce5f Mon Sep 17 00:00:00 2001
Message-Id: <9a8120a8d7eb872da43e61d598c226f0d6b7ce5f.1649664304.git.geliang.tang@suse.com>
From: Menglong Dong <dong.menglong@zte.com.cn>
Date: Tue, 5 Jan 2021 07:20:47 -0800
Subject: [PATCH] selftests/bpf: Remove duplicate include in test_lsm
Git-commit: 9a8120a8d7eb872da43e61d598c226f0d6b7ce5f
Patch-mainline: v5.12-rc2
References: bsn#127
Modified-by-SEL: No


'unistd.h' included in 'selftests/bpf/prog_tests/test_lsm.c' is
duplicated.

Signed-off-by: Menglong Dong <dong.menglong@zte.com.cn>
Signed-off-by: Andrii Nakryiko <andrii@kernel.org>
Acked-by: Martin KaFai Lau <kafai@fb.com>
Link: https://lore.kernel.org/bpf/20210105152047.6070-1-dong.menglong@zte.com.cn
Signed-off-by: Alexei Starovoitov <ast@kernel.org>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 tools/testing/selftests/bpf/prog_tests/test_lsm.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tools/testing/selftests/bpf/prog_tests/test_lsm.c b/tools/testing/selftests/bpf/prog_tests/test_lsm.c
index 6ab29226c99b..2755e4f81499 100644
--- a/tools/testing/selftests/bpf/prog_tests/test_lsm.c
+++ b/tools/testing/selftests/bpf/prog_tests/test_lsm.c
@@ -10,7 +10,6 @@
 #include <unistd.h>
 #include <malloc.h>
 #include <stdlib.h>
-#include <unistd.h>
 
 #include "lsm.skel.h"
 
-- 
2.34.1

